.github/ISSUE_TEMPLATE/new_component.yaml

```diff
@@ -26,8 +26,9 @@ body:
       label: Is this a vendor-specific component?
       description: A vendor-specific component directly interfaces with a vendor-specific API and is expected to be maintained by a representative of the same vendor.
       options:
-        - label: This is a vendor-specific component
-        - label: If this is a vendor-specific component, I am proposing to contribute this as a representative of the vendor.
+        - label: This is a vendor-specific component.
+        - label: (If a vendor-specific component) I am proposing to contribute and maintain this as a representative of the vendor.
+        - label: (If a vendor-specific component) I am a member of the OpenTelemetry project.
```

Comment:
I'm in favor of moving this check out of the `vendor-specific component` section and into a general section of the template.  I like the idea of asking up front if the contributor is a member or not, regardless of whether they are sponsoring a vendor-specific component.

Reply:
I am of two minds on this one. The question to me is whether or not we should assume that the proposer of the component is the contributor of it. In practice, I think this is the case the vast majority of the time, but I think there might be some value in welcoming proposals from end users, non-code contributors, etc, which could then be picked up by a contributor.

For vendor-specific components, we've made this quite explicit so I don't think it's a problem.

Reply:
I wonder if being a member of the OTel community to be a code owner should be explained somewhere in a separate section in this template, to help newcomers understand why it's important to their contribution.

Reply:
> Is this checkbox needed at all? GitHub can already tell us whether they are a member of the otel project

The case for it would be so that the proposer understands the requirements.

Reply:
How about "I have identified one or more people who are members or aspiring members of the OpenTelemetry project who can volunteer to serve as codeowners, and am listing them in this proposal for confirmation."

Reply:
> How about "I have identified one or more people who are members or aspiring members of the OpenTelemetry project who can volunteer to serve as codeowners, and am listing them in this proposal for confirmation."

This sounds good. I suppose a required text field is appropriate then.

Reply:
I've reworked the PR to just add a new "Code Owners" section alongside the "Sponsor" section.